.github/workflows/sbom-secscan.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
               path: scanner
               persist-credentials: false
           - name: Install uv
-            uses: astral-sh/setup-uv@d0cc045d04ccac9d8b7881df0226f9e82c39688e  # v6.8.0
+            uses: astral-sh/setup-uv@85856786d1ce8acfbcc2f13a5f3fbd6b938f9f41  # v7.1.2
```

Comment:
For this one we need to make sure that the hosted runners have the newer node available.

Reply:
Apparently they pull in the latest images every ~8h